backend/app/services/adapters/executor_kinds.py

```
@@ -2670,5 +2720,41 @@ async def delete_executor_task_async(
                 status_code=500, detail=f"Error deleting executor task: {str(e)}"
             )
 
+    def get_executor_address_sync(
+        self, executor_name: str, executor_namespace: str = ""
```

Ruff (0.14.14)

[warning] Unused method argument: `executor_namespace`

(ARG002)